pkg/ovs/openflow/ofctrl_packetout.go

```diff
@@ -298,6 +298,70 @@ func (b *ofPacketOutBuilder) AddLoadRegMark(mark *RegMark) PacketOutBuilder {
 	return b.AddLoadAction(name, uint64(mark.value), mark.field.rng)
 }
 
+func (b *ofPacketOutBuilder) AddResubmitAction(inPort *uint16, table *uint8) PacketOutBuilder {
+	act := ofctrl.NewResubmit(inPort, table)
+	b.pktOut.Actions = append(b.pktOut.Actions, act)
+	return b
+}
+
+type PacketOutType int
+
+const (
+	// RejectPodLocal represents this packetOut is used to reject Pod-to-Pod traffic
+	// and for this response, the srcPod and the dstPod are on the same Node.
+	RejectPodLocal PacketOutType = iota
```

Comment:
this is very specialized and only used for reject packets, so why is the type name something as generic as `PacketOutType`?

Reply:
I plan to use it as a generic type. But I only included rejecting cases for now.

Reply:
Changed.